### ISSUE TITLE
Consider integrating Fotorama.io slideshow/viewer - touch-device support

### ISSUE THREAD

**Issue by osos** [owncloud/gallery#94]
Please consider to integrate the www.fotorama.io slideshow. It is a jquery gallery.

It has support for touch devices as well.
##

**Comment by oparoz** [comment]
The one thing I see it has missing is zooming capabilities, but it does have fullscreen mode.
Ideally, this should be supported in parallel and users could select the one they prefer to use, because I only see this working well for pictures, not necessarily for other image types.

**Comment by setnes** [comment]
PhotoSwipe might be an option for this as well.  It's simple, clean, and looks great on mobile devices.

http://photoswipe.com/

**Comment by oparoz** [comment]
Thanks, it could be a viable replacement as it supports most features we already use and is available via bower.

**Comment by setnes** [comment]
Maybe I should have opened a new issue.  Osos was specifically requesting Fotorama.  I'd like to be less specific and just request that when viewing photos on a mobile device the user is allowed to swipe through the photos instead of clicking on the prior and next icons.  I'm not sure if it matters if it is something like PhotoSwipe, Photorama, or enhancements to the existing code.  Another issue that might be corrected with PhotoSwipe or Photorama is that rotating a phone or resizing any browser window doesn't result in the image being corrected to match the new size or orientation.

**Comment by oparoz** [comment]
Yes, please open 2 new issues. One for the swipe and one for the rotation :)

**Comment by setnes** [comment]
Done.  :)

Swipe Issue:  https://github.com/owncloud/galleryplus/issues/141
Resize/Rotate Issue:  https://github.com/owncloud/galleryplus/issues/142

**Comment by oparoz** [comment]
Thanks!

**Comment by oparoz** [comment]
This issue was moved to nextcloud/gallery#23
